### ISSUE TITLE
bazel + protoc 3.1.0 (3.1.1): php generator not found

### ISSUE THREAD

**Issue by pcj** [google/protobuf#2221]
This seems to be blocking me from upgrading protoc to post 3.0.0.  Building with bazel, protoc compiles successfully, but I get the following error when trying to invoke it:

```sh
dyld: Symbol not found: __ZTVN6google8protobuf8compiler3php9GeneratorE
```

```sh
  Referenced from: /private/var/tmp/_bazel_pcj/63330772b4917b139280caef8bb81867/execroot/rules_protobuf/bazel-out/host/bin/external/com_github_google_protobuf/protoc
  Expected in: flat namespace
 in /private/var/tmp/_bazel_pcj/63330772b4917b139280caef8bb81867/execroot/rules_protobuf/bazel-out/host/bin/external/com_github_google_protobuf/protoc
```

I'm not referencing or using the new php generator however.

Not sure how to fix this.  Perhaps the `BUILD` file needs to be updated?

**Comment by xfxyjwf** [comment]
Yes, the build file needs to be updated. Could you help send us a pull request to fix the issue?

**Comment by TeBoring** [comment]
Fixed in 3.1.x branch. Will merge it to master later this week.

**Comment by pcj** [comment]
Thanks @TeBoring @xfxyjwf.  The latest commit on 3.1.x 52ab3b07ac9a6889ed0ac9bf21afd8dab8ef0014 fixes this on my end.  I'm going to use that commit for now, looking forward to merge to master.
